Automattic/Edit-Flow
# Add PHP 8.2 fixes

## Description

Alternative to #699 

## Steps to Test

Check QM and php logs to ensure no more PHP 8.2 dynamic property deprecation notices are being set.
